oracle/src/lib.rs

```diff
@@ -415,6 +424,18 @@ impl Oracle {
         result
     }
 
+    pub fn get_nonce(&self, account: AccountId) -> Option<U128> {
+        self.nonces.get(&account)
+    }
+
+    pub fn get_nonces(&self) -> HashMap<AccountId, U128> {
```

Comment:
I think typically we want to have paging or a `max` value
Not sure that I understand when this will be called?

Reply:
This is O(n) where n is the number of clients. I do not expect that many clients to worry about this.

This is called on Chainlink initiator start, so that specific node does not duplicate requests that could already be raised. On start, we check the latest state and the node will raise requests from that point forward, not for already existing requests.